stl/inc/random

```diff
@@ -901,15 +902,16 @@ public:
 };

```

Comment:
Above, I think we should change `subtract_with_carry() : _Mybase(default_seed) {}` to use `0u` for consistency.